.github/.workflows/cicd.yml

```diff
@@ -0,0 +1,57 @@
+# .github/workflows/github-pages.yml
```

Comment:
either change this to /cicd.yml or change the filename to /github-pages.yml (i think the second option is clearer)